### PR TITLE
add ImageBlock component

### PR DESCRIPTION
**Pull request by ziggabyte**
This PR adds an ImageBlock component to the cms and frontend, intended to be used by editors to add images between TextBlock-components

### CODE REVIEW

File: frontend/src/components/ImageBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        return { height: 300, width: 450 };
+      default:
+        return { height: 200, width: 300 };
+    }
+  };
+
+  const imageSize = setImageSize();
+
+  return (
+    <Box>
```
Comment (Bagera) [inline frontend/src/components/ImageBlock.tsx:25]: I think a figure with a figcaption is a better choice of elements here.

**Review by Bagera — CHANGES_REQUESTED** [review]
We should have more semantic elements than div and p here

File: frontend/src/components/ImageBlock.tsx (hunk 1 above)
Comment (ziggabyte) [reply frontend/src/components/ImageBlock.tsx:25]: Will exchange! Thank you!

File: frontend/src/components/ImageBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import Image from 'next/image';
+import getCmsMedia from '../utils/getCmsMedia';
+
+const ImageBlock = ({ ...rest }) => {
+  const { description, image, size } = rest;
+
+  const setImageSize = () => {
```
[the remark this answers is not in the document]
Comment (ziggabyte) [reply frontend/src/components/ImageBlock.tsx:7]: In the conversation with Robin and Richard it was requested that the editor could decide the size of the displayed image in a dropdown in the cms, and this is my interpretation of how to take the value the editor decided and adjust the size of image from that.

[the remark this answers is not in the document]
Comment (Bagera) [reply frontend/src/components/ImageBlock.tsx:7]: I see. I just don't like the random values used. Is it supposed to crop the images? Else I think we should keep the the aspect ratio of the original images.

I think that if we could make better guesses on the sizes.

[the remark this answers is not in the document]
Comment (ziggabyte) [reply frontend/src/components/ImageBlock.tsx:7]: So I put original image size in all until we have better values to put into it?

[the remark this answers is not in the document]
Comment (Bagera) [reply frontend/src/components/ImageBlock.tsx:7]: Sure, that might be better for now.

**Review by Bagera — APPROVED** [review]

**Pull request merged**